src/background/Wallet/model/types.ts

```diff
@@ -45,6 +45,7 @@ interface PublicPreferences {
   configurableNonce?: boolean;
   invitationBannerDismissed?: boolean;
   recentAddresses?: string[];
+  recentSearch?: string[];
```

Comment:
This shouldn't be part of wallet record; instead it should be a separate storage entry, like `chain-configs`